koku/api/report/view.py

```diff
@@ -116,7 +119,13 @@ class ClassMapper(object):
                     'serializer': OCPAWSQueryParamSerializer,
                     'query_handler': OCPAWSReportQueryHandler,
                     'tag_handler': [AWSTagsSummary]
-                }
+                },
+                {
+                    'report': 'tags',
+                    'serializer': OCPAWSTagsQueryParamSerializer,
+                    'query_handler': OCPAWSTagQueryHandler,
+                    'tag_handler': [AWSTagsSummary, OCPUsagePodLabelSummary]
```

Comment:
Right now we only look at the filtered subset of AWSTags that matched, so the handler field can drop OCP Usage labels. 

```suggestion
                    'tag_handler': [AWSTagsSummary]
```

Reply:
I've pushed this fix.